101-recovery-services-vm-workload-backup/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vaultName": {
+            "value": "SqlBackupDemoVault"
+        },
+        "vmName": {
+            "value": "GET-PREREQ-virtualMachineName"
+        },
+        "policyName": {
```

Comment:
remove these non-GEN values and use defaultValues in the template

Reply:
Done.

Reply:
I still see all the literal values here

Reply:
Oh yes, missed this reference. Fixed it now.

Reply:
Still have the non-GEN values in here that need to be remoed

Reply:
This is from older commit. Latest commit has below value
 "vmName": {
            "value": "sqlvmbackupdemo"
}

Reply:
What I see in the branch for this PR are non-GEN values.  These should be defaultValues in the template and removed from the parameters file.